subgraph/schema.graphql

```diff
@@ -147,21 +152,15 @@ type DigitalaxMarketplaceOffer @entity {
 
 type DigitalaxMarketplacePurchaseHistory @entity {
   id: ID!
+  eventName: String!
+  timestamp: BigInt!
   transactionHash: Bytes!
   token: DigitalaxGarment!
   buyer: Bytes!
   value: BigInt!
   isPaidWithMona: Boolean!
   monaTransferredAmount: BigInt!
   garmentAuctionId: BigInt!
-  discount: BigInt!
```

Comment:
I will probably have to bring this back because of the new contract changes. But it is fine for now